### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by udaycruise2903**
Hi @chshersh 

I followed up your discussion in Issues section. 

Any comments on improvements?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,111 +1,48 @@
-# co-log
+# co-log-polysemy
 
 ![Co-logo](https://user-images.githubusercontent.com/8126674/80955687-92f21a80-8df7-11ea-90d3-422dafdc8391.png)
 
 [![GitHub CI](https://github.com/kowainik/co-log/workflows/CI/badge.svg)](https://github.com/kowainik/co-log/actions)
```
Comment (chshersh) [inline README.md]: A link here needs to be changed to this repository

File: README.md [hunk 2]
```diff
@@ -1,111 +1,48 @@
...
-[![Build status](https://img.shields.io/travis/kowainik/co-log.svg?logo=travis)](https://travis-ci.org/kowainik/co-log)
-[![Windows build](https://ci.appveyor.com/api/projects/status/github/kowainik/co-log?branch=main&svg=true)](https://ci.appveyor.com/project/kowainik/co-log)
+[![Hackage][hk-img-ps]][hk-ps]
```
Comment (chshersh) [inline README.md:6]: It looks like `hk-img-ps` and `hk-ps` links were deleted 🤔 
You can either return them back on the bottom of this README or inline them directly here 🙂

File: README.md [hunk 3]
```diff
@@ -1,111 +1,48 @@
...
+[co-log-core](http://hackage.haskell.org/package/co-log-core) logging based on 
+the [polysemy](http://hackage.haskell.org/package/polysemy) extensible 
+effects library.
+
+## Goals
+
+We created the `co-log-polysemy` project in pursuit of the following goals:
+
+## Features
+
```
Comment (chshersh) [inline README.md]: I think we can skip this part for now 👍🏻 
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,111 +1,48 @@
...
+will need to set it up with the these easy steps:
+
+1. Add the dependency on `co-log-polysemy` in your project's
+   `.cabal` file. For this, you should modify the `build-depends`
+   section by adding the name of this library. After the adjustment,
+   this section could look like this:
+
+   ```haskell
+   build-depends: base >= 4.11 && < 4.16
+                , co-log-polysemy ^>= 0.0.1.3
```
Comment (chshersh) [inline README.md]: Let's generic names here so we need to update fewer places when we bump up dependencies
```suggestion
   build-depends: base ^>= LATEST_SUPPORTED_BASE
                , co-log-polysemy ^>= LATEST_VERSION
```

File: README.md [hunk 5]
```diff
@@ -1,111 +1,48 @@
...
+## Goals
+
+We created the `co-log-polysemy` project in pursuit of the following goals:
+
+## Features
+
+## How to use
+
+`co-log-polysemy` is compatible with the GHC compiler
+versions starting from `8.4.4` upto `9.0.1`.
```
Comment (chshersh) [inline README.md]: I'm thinking how to remove specific versions from here as well 🤔 
Maybe just give a link to this page that shows all supported GHC versions?

* https://matrix.hackage.haskell.org/#/package/co-log-polysemy

File: README.md [hunk 6]
```diff
@@ -1,111 +1,48 @@
...
+2. To use the module add the import:
+
+   ```haskell
+   import Colog.Polysemy
+   ```
+
+## Comparison to other libraries
+
+## Acknowledgement
+
```
Comment (chshersh) [inline README.md]: Let's remove this part as well
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -1,111 +1,48 @@
...
+   ```haskell
+   build-depends: base >= 4.11 && < 4.16
+                , co-log-polysemy ^>= 0.0.1.3
+   ```
+
+2. To use the module add the import:
+
+   ```haskell
+   import Colog.Polysemy
+   ```
```
Comment (chshersh) [inline README.md]: Let's add a complete example of the library usage with imports and `main` function 🙂 
You can take the example from here:

https://github.com/co-log/co-log-polysemy/blob/main/example/Main.hs

**Review by chshersh — APPROVED** [review]
@udaycruise2903 Thanks for your contribution! 🤗 
I left some comments with suggestions and further improvements.
But it's a great start 👍🏻

File: README.md (hunk 7 above)
Comment (udaycruise2903) [reply README.md]: If the example is not documented in the right way. please suggest the changes.

**Review by chshersh — APPROVED** [review]
@udaycruise2903 Thanks! I think it looks good for now 🙂

**Pull request merged**